packages/preview/twilight-book/0.1.6/src/book.typ

```diff
@@ -0,0 +1,310 @@
+#import "../themes/index.typ" : * // 导入主题设置模块 / Import theme settings module
+
+#import "models.typ" : * // 导入模块 / Import models module
+
+// 用于弥补缺少 `std` 作用域的工作区。
+// Workaround for missing `std` scope in workspace.
```

Comment:
This solution works fine, but for info it is possible to write `std.bibliography`, the `std` scope does exist.